tests/federation/test_federation_catch_up.py

```diff
@@ -431,28 +431,22 @@ def test_catch_up_on_synapse_startup(self) -> None:
         # ACT: call _wake_destinations_needing_catchup
 
         # patch wake_destination to just count the destinations instead
-        woken = []
+        woken = set()
 
         def wake_destination_track(destination: str) -> None:
-            woken.append(destination)
+            woken.add(destination)
 
         self.federation_sender.wake_destination = wake_destination_track  # type: ignore[assignment]
 
-        # cancel the pre-existing timer for _wake_destinations_needing_catchup
-        # this is because we are calling it manually rather than waiting for it
-        # to be called automatically
-        assert self.federation_sender._catchup_after_startup_timer is not None
-        self.federation_sender._catchup_after_startup_timer.cancel()
-
-        self.get_success(
-            self.federation_sender._wake_destinations_needing_catchup(), by=5.0
-        )
+        self.pump(by=5.0)
```

Comment:
Does this magic value pertain to anything? We should comment why we wait this long

I'm guessing this is something to do with `WAKEUP_INTERVAL_BETWEEN_DESTINATIONS_SEC` but I would expect to pump longer in order to wake up all 26 destinations.

Reply:
I am guessing it's related to this `* 100` here: https://github.com/matrix-org/synapse/blob/develop/tests/unittest.py#L619.

I am going to add a comment.